src/GoTrueClient.ts

```diff
@@ -240,7 +241,10 @@ export default class GoTrueClient {
           scopes: options.scopes,
         })
       }
-      throw new Error(`You must provide either an email, phone number or a third-party provider.`)
+      if (openIDConnect) {
+        return this._handleOpenIDConnectSignIn(openIDConnect)
+      }
+      throw new Error(`You must provide either an email, phone number or a third-party provider or OpenID Connect.`)
```

Comment:
```suggestion
      throw new Error(`You must provide either an email, phone number, a third-party provider or OpenID Connect.`)
```

Is it clear what ` OpenID Connect` means?